### PR TITLE
CLN: remove fastparquet compat shim

### PR DESCRIPTION
**Pull request by jbrockmendel**
Introduced in #19434 Jan 2018.

### CODE REVIEW

File: pandas/core/internals/blocks.py [hunk 1]
```diff
@@ -3040,8 +3028,7 @@ def get_block_type(values, dtype=None):
     return cls
 
 
-def make_block(values, placement, klass=None, ndim=None, dtype=None,
-               fastpath=None):
+def make_block(values, placement, klass=None, ndim=None, dtype=None):
```
Comment (jreback) [inline pandas/core/internals/blocks.py]: if you can type things would be great (new PR)

**Comment by jreback** [comment]
lgtm.

**Comment by jreback** [comment]
I guess fastparquet 0.2.1 still has the offending code. Can you add an issue to their tracker? (this is the failing build)

**Comment by jbrockmendel** [comment]
> Can you add an issue to their tracker?

Looks like Tom beat me to it by about 2 weeks

**Comment by TomAugspurger** [comment]
I'll push a fix for the issue.

**Comment by TomAugspurger** [comment]
I'd recommend skipping for fastparquet <0.3.2

**Comment by jreback** [comment]
> I'd recommend skipping for fastparquet <0.3.2
> […](#)

we can’t skip as that just papers over; we would need to bump or not merge this patch

**Comment by jorisvandenbossche** [comment]
Let's leave this for after 0.25.0 then, or at least until there is a new fastparquet release with the fix. It's a bit unfortunate that fastparquet didn't fix that a year ago.

**Comment by TomAugspurger** [comment]
Waiting may be prudent. I'm going to try to do a FP release today.

**Comment by jbrockmendel** [comment]
@TomAugspurger should I stick a pin in this for [?] weeks?

**Comment by jreback** [comment]
yeah I think we need FP release around for a while before we can do this.

**Comment by jbrockmendel** [comment]
Closing to clear the queue, will revisit a release or two from now

**Pull request closed without merging**